packages/databyss-data/pouchdb/entries/lib/setBlockRelations.ts

```diff
@@ -1,44 +1,69 @@
-import { BlockRelationPayload } from '@databyss-org/editor/interfaces'
+import {
+  BlockRelationPayload,
+  BlockRelationResponse,
+  BlockRelationOperation,
+} from '@databyss-org/editor/interfaces'
+// import { DocumentType } from '../../interfaces';
```

Comment:
Please do not commit commented-out code. They just clutter up the codebase and are very hard to find and remove en-masse later on